### ISSUE TITLE
PC editor needs option to swap out pets in middle of missions

### ISSUE THREAD

**Issue by Xanthor** [robot9706/FSSE#113]
I noticed when you edit a dweller thats out in the fields that you can add items for healing and curing, but nothing to swap out the pet till they return. When game updated, it reverted all my pets from 50k to 58 and was not able to fix this during a mission I was in. To be able to edit the stats in middle of mission, would be a fantastic feature.

**Comment by robot9706** [comment]
Changing a pet of a dweller which is on a quest doesn't actually change the pet? I need to check that out.

**Comment by Xanthor** [comment]
I discovered the issue after the game loaded and found out on the PC they updated. When the update loaded, it changed all 250 pets back to their normal stats and once I updated the new mods, had to change them all back and realized the characters on a quest that was saved in the middle of the quest, their pets would not update. Also, you cant change anything on any character that's in the middle of a quest. Like healing or cure items, or even if one character out of the 3 dies, you cant resurrect them till you either finish the quest or abort the quest. That's on the PC, I never checked if the same with android.

**Comment by robot9706** [comment]
The pets reverted back to their original values because the game checks those. A mod allows editing them so I understand that part. So I guess dwellers are moved somewhere else when they go on missions, that's why changes don't apply.

**Comment by Xanthor** [comment]
I been looking throughout the data area to see if I could find where those quests saved so I can manually hex edit items in on an open mission. Nothing to do with your program, but for some reason other than forgetting to add supplies before setting out, sometimes I load back into the quest I saved from to find 0 supplies for healing or poison. Last night watched one of my missions go south when two character's died. That's when I found out their pets stopped doing 50 to 70k damage and had reverted back.
